lib/TutorialServo/src/TutorialServo.cpp

```diff
@@ -0,0 +1,25 @@
+#include "TutorialServo.h"
+#include "mbed.h"
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees, 
+    float minPulsewidthInMs, float maxPulsewidthInMs):
+    m_servoPwmOut(servoPin), 
+    m_servoRangeInDegrees(servoRangeInDegrees), 
+    m_minPulsewidthInMs(minPulsewidthInMs), 
+    m_maxPulsewidthInMs(maxPulsewidthInMs){}
+
+void TutorialServo::setPositionInDegrees(float degrees){
+    float PulsewidthRangeInMs = m_maxPulsewidthInMs - m_minPulsewidthInMs;
+    float new_pulse_width = (degrees/m_servoRangeInDegrees)*PulsewidthRangeInMs + m_minPulsewidthInMs;
+    m_servoPwmOut.pulsewidth(new_pulse_width);
+}
+
+float TutorialServo::getServoRangeInDegrees( ) const{
+    return m_servoRangeInDegrees;l
```

Comment:
typo

Reply:
Thanks, is it possible to fix the clang-format stuff now? I tried installing it but I still get the error